plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/core.py

```diff
@@ -257,7 +282,13 @@ def sweep(self, arguments: List[str]) -> None:
         parsed = parser.parse_overrides(arguments)
 
         for override in parsed:
-            params[override.get_key_element()] = make_nevergrad_parameter(override)
+            if override.key_or_group == "batch_size":
+                import pdb;pdb.set_trace()
```

Comment:
oops?

Reply:
Also, try to use your IDE for debugging (pycharm works well).

Reply:
I think this is outdated code :) 

Sounds good! i will check out pycharm debugging!